components/Navigation.js

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components";
+import Link from "next/link";
+const StyledNavigation = styled.nav`
+  background-color: grey;
+  padding: 10px;
+`;
```

Comment:
I suggest to stick the navigation to the bottom or top, so that it is always visible.